src/demo-app/accessibility/accessibility.html

```diff
@@ -0,0 +1,36 @@
+<md-sidenav-container class="demo-root" fullscreen>
```

Comment:
Changed to `<nav>` with `<a>` but "skip link" seems not working https://github.com/angular/angular/issues/6595